pages/notifications/manage-notifications/index.tsx

```diff
@@ -413,4 +419,29 @@ const ManageNotifications = (props) => {
   );
 };
 
+const checkShouldHideConfirmationMessage = (
+  migrationBannerProps: {
+    migrationType: string;
+    applyMigrationStatus: string;
+    findMigrationStatus: string;
+  },
+  shouldRenderMigrationBanner: boolean,
+) => {
+  const isSubscriptionNotificationMigration =
+    migrationBannerProps.migrationType ===
+    LOGIN_NOTICE_TYPES.SUBSCRIPTION_NOTIFICATIONS;
+
+  if (!shouldRenderMigrationBanner && isSubscriptionNotificationMigration)
+    return false;
+
+  const oneOfTheMigrationsPassed =
```

Comment:
nitpicking but this being `true` could also mean both migrations passed

```suggestion
  const migrationPassed =
```

Reply:
I almost went with `atLeastOneOfThe...`

Reply:
Have renamed that